app/src/main/java/ingsw/group1/findmyphone/fragment/LogFragment.java

```diff
@@ -3,31 +3,37 @@
 import android.content.Context;
 import android.os.Bundle;
 import android.view.LayoutInflater;
+import android.view.MenuItem;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ImageButton;
+import android.widget.PopupMenu;
+import android.widget.SearchView;
 
 import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
-import java.util.List;
-
 import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.event.EventOrder;
 import ingsw.group1.findmyphone.event.SMSLogDatabase;
-import ingsw.group1.findmyphone.event.SMSLogEvent;
-import ingsw.group1.findmyphone.log.LogItem;
 import ingsw.group1.findmyphone.log.LogItemFormatter;
+import ingsw.group1.findmyphone.log.LogManager;
 import ingsw.group1.findmyphone.log.LogRecyclerAdapter;
 
 /**
  * Fragment holding the Log screen. Contains a list of the Log events.
  *
  * @author Riccardo De Zen.
  */
-public class LogFragment extends Fragment {
+public class LogFragment extends Fragment implements PopupMenu.OnMenuItemClickListener {
 
-    private List<LogItem> logItemList;
+    private LogManager logManager;
+    private RecyclerView logRecycler;
+    private ImageButton sortButton;
+    private SearchView searchView;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'searchView' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5028692)